src/components/Tree/Node.tsx

```diff
@@ -141,27 +135,33 @@ export const Node = ({
             >
                 <div
                     className={merge([
-                        ' tw-flex tw-py-1 tw-px-2 tw-rounded tw-no-underline tw-leading-6',
+                        ' tw-flex tw-py-2 tw-px-2.5 tw-no-underline tw-leading-5',
                         strong && 'tw-font-bold',
-                        value && !selected && 'hover:tw-bg-black-5',
-                        selected ? 'tw-bg-violet-60 tw-text-white' : 'tw-text-black',
+                        value && !selected && 'hover:tw-bg-black-10 hover:tw-text-box-neutral-inverse-hover',
+                        selected
+                            ? 'tw-bg-violet-60 tw-text-white hover:tw-bg-box-selected-strong-hover hover:tw-text-box-selected-strong-inverse-hover'
+                            : 'tw-text-black',
                     ])}
                     onMouseEnter={setHoveredTrue}
                     onMouseLeave={setHoveredFalse}
                 >
                     <a
                         data-test-id="node-link"
-                        className="tw-flex tw-items-center tw-flex-grow tw-justify-between tw-cursor-pointer"
+                        className={merge([
+                            'tw-flex tw-items-center tw-flex-grow tw-justify-between tw-cursor-pointer',
+                            parentIds.length === 1 && 'tw-pl-4',
+                            parentIds.length > 1 && 'tw-pl-8',
+                        ])}
                         aria-selected={selected}
                         onClick={onNodeClick}
                     >
-                        <div className="tw-flex tw-space-x-2 tw-items-center">
-                            <span data-test-id="toggle" onClick={toggleNodesVisibility}>
+                        <div className="tw-flex tw-space-x-1 tw-items-center">
+                            <span data-test-id="toggle" className="tw-w-3" onClick={toggleNodesVisibility}>
                                 {nodes &&
                                     (showNodes ? (
-                                        <IconCaretDown size={IconSize.Size16} />
+                                        <IconCaretDown size={IconSize.Size12} />
```

Comment:
Hey @marguslant0, Looking at the design, I don't think we should use the CaretDown Icon here. It should be 9px high and not 12 and completely filled. It would be better to make it custom, don't you think?

Reply:
Hey @imoutaharik, thanks for the suggestion! Do you know if we have such custom triangle already in use somewhere in Arcade?

Reply:
No, apparently all collapsable components in Arcade are using the Caret Icon. It seems only this one has a slightly different design.

Reply:
Perhaps it's new the new intended design for all carets? If so, maybe it makes sense to keep it as is for now, for consistency?

Reply:
I would totally get @thisguyscode's input on this. 😉 Let's not decide on the design ourselves.

Reply:
We'll discuss it next week. I'll approve and merge this version for the urgency.